### ISSUE TITLE
TypedRelation service call in Alchemy Language Java SDK not returning entities

### ISSUE THREAD

**Issue by nastacio** [watson-developer-cloud/java-sdk#367]
I am using the [AlchemyLanguage] service, more specifically the Typed Relations service.

I am using version 3.0.2-SNAPSHOT

        <repository>
            <id>sonatype</id>
            <url>https://oss.sonatype.org/content/repositories/snapshots</url>
        </repository>

        <dependency>
            <groupId>com.ibm.watson.developer_cloud</groupId>
            <artifactId>java-sdk</artifactId>
            <version>3.0.2-SNAPSHOT</version>
        </dependency>

Somehow the resulting Typed Relation objects do not contain the "entities" values, just the relation type:

   ```
 @Test
    public void testX() throws IOException {
        AlchemyLanguage service = new AlchemyLanguage();
        service.setApiKey("...");
        service.setEndPoint("https://gateway-a.watsonplatform.net/calls");

        Path input = Paths.get("./src/test/resources/9563489.txt");
        ByteArrayOutputStream baos = new ByteArrayOutputStream();
        Files.copy(input, baos);

        Map<String, Object> params = new HashMap<String, Object>();
        params.put(AlchemyLanguage.TEXT, baos.toString("UTF-8"));
        params.put(AlchemyLanguage.MODEL_ID, "ie-en-news");

        ServiceCall<TypedRelations> trCall = service.getTypedRelations(params);
        TypedRelations typedRelations = trCall.execute();
        List<TypedRelation> trs = typedRelations.getTypedRelations();
        trs.stream().forEach(tr -> {
            System.out.println(tr.getEntities());
            System.out.println("\t" + tr.getType());
        });
    }
```

And the output is:

```
null
	locatedAt
null
	hasAttribute
null
	partOfMany

```

**Comment by nastacio** [comment]
In case it helps (still sorting out issues with my local git repo, will try and submit a pull request later)

Code fix
[Archive.zip](https://github.com/watson-developer-cloud/java-sdk/files/344615/Archive.zip)

Testcase fix.
[AlchemyLanguageIT.java.zip](https://github.com/watson-developer-cloud/java-sdk/files/344619/AlchemyLanguageIT.java.zip)

**Comment by germanattanasio** [comment]
fixed in #403
